src/components/Banner/Banner.scss

```diff
@@ -157,11 +153,6 @@ $secondary-action-horizontal-padding: 1.5 * spacing(tight);
 
   .Ribbon {
     padding-right: $intermediate-spacing;
-
-    // stylelint-disable-next-line selector-max-combinators, selector-max-class
-    .newDesignLanguage & {
-      padding-right: spacing();
-    }
```

Comment:
I don't think this worked because that's not the right syntax so just cleaning up a bit, same styling was applied [here](https://github.com/Shopify/polaris-react/blob/0daad919bcedebcd99cf27be1cea97c666543e01/src/components/Banner/Banner.scss#L146) as part of https://github.com/Shopify/polaris-react/pull/3355